### PR TITLE
Editorial: remove Type() macro

### PR DESCRIPTION
**Pull request by ljharb**
Closes #3339.

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -20684,39 +20683,53 @@ <h1>
...
+                  <tr><td> `&lt;&lt;`     </td><td> BigInt::leftShift          </td></tr>
+                  <tr><td> `&gt;&gt;`     </td><td> BigInt::signedRightShift   </td></tr>
+                  <tr><td> `&gt;&gt;&gt;` </td><td> BigInt::unsignedRightShift </td></tr>
+                  <tr><td> `&amp;`        </td><td> BigInt::bitwiseAND         </td></tr>
+                  <tr><td> `^`            </td><td> BigInt::bitwiseXOR         </td></tr>
+                  <tr><td> `|`            </td><td> BigInt::bitwiseOR          </td></tr>
+                </tbody>
+              </table>
+            </figure>
+        1. else if _lNum_ is a Number, then:
```
Comment (michaelficarra) [inline spec.html:20709]:
```suggestion
        1. Else,
          1. Assert: _lNum_ is a Number.
```

File: spec.html [hunk 2]
```diff
@@ -20684,39 +20683,53 @@ <h1>
         1. Let _rNum_ be ? ToNumeric(_rVal_).
         1. If SameType(_lNum_, _rNum_) is *false*, throw a *TypeError* exception.
         1. If _lNum_ is a BigInt, then
-          1. If _opText_ is `**`, return ? BigInt::exponentiate(_lNum_, _rNum_).
-          1. If _opText_ is `/`, return ? BigInt::divide(_lNum_, _rNum_).
-          1. If _opText_ is `%`, return ? BigInt::remainder(_lNum_, _rNum_).
-          1. If _opText_ is `>>>`, return ? BigInt::unsignedRightShift(_lNum_, _rNum_).
```
[the remark this answers is not in the document]
Comment (ljharb) [reply spec.html:20690]: ahhhh gotcha, ok

File: spec.html [hunk 3]
```diff
@@ -20688,35 +20687,50 @@ <h1>
...
+                </thead>
+                <tbody>
+                  <tr><td> `*`            </td><td> BigInt::multiply           </td></tr>
+                  <tr><td> `+`            </td><td> BigInt::add                </td></tr>
+                  <tr><td> `-`            </td><td> BigInt::subtract           </td></tr>
+                  <tr><td> `&lt;&lt;`     </td><td> BigInt::leftShift          </td></tr>
+                  <tr><td> `&gt;&gt;`     </td><td> BigInt::signedRightShift   </td></tr>
+                  <tr><td> `&amp;`        </td><td> BigInt::bitwiseAND         </td></tr>
+                  <tr><td> `^`            </td><td> BigInt::bitwiseXOR         </td></tr>
+                  <tr><td> `|`            </td><td> BigInt::bitwiseOR          </td></tr>
```
Comment (michaelficarra) [inline spec.html:20705]: Let's tighten up the whitespace padding on these columns.

Comment (ljharb) [reply spec.html:20705]: that'd be automatic if i remove the formatting ignore, but it looks much worse when i do that, which is why i left it in place from before

**Review by michaelficarra — APPROVED** [review]
LGTM

File: spec.html (hunk 3 above)
Comment (michaelficarra) [reply spec.html:20705]: No I don't mean all the way, I just mean the excess width where every column has too much.

Comment (ljharb) [reply spec.html:20705]: ah ok gotcha, on it

File: spec.html [hunk 4]
```diff
@@ -1121,7 +1121,6 @@ <h1>Identity</h1>
 <emu-clause id="sec-ecmascript-data-types-and-values" aoid="Type">
```
Comment (michaelficarra) [inline spec.html:1121]:
```suggestion
<emu-clause id="sec-ecmascript-data-types-and-values">
```

File: spec.html [hunk 5]
```diff
@@ -1118,10 +1118,9 @@ <h1>Identity</h1>
   </emu-clause>
 </emu-clause>
 
-<emu-clause id="sec-ecmascript-data-types-and-values" aoid="Type">
+<emu-clause id="sec-ecmascript-data-types-and-values">
```
Comment (syg) [inline spec.html:1121]: We should have an `oldid` somewhere, maybe on SameType, as a hint to readers for the new style.

**Review by syg — APPROVED** [review]
lgtm % oldid comment

File: spec.html (hunk 5 above)
Comment (michaelficarra) [reply spec.html:1121]: Actually this doesn't create its own anchor, the call links directly to `#sec-ecmascript-data-types-and-values`, so we should be good.

**Pull request merged**